components/DatePicker/DatePicker.jsx

```diff
@@ -91,51 +100,58 @@ export default class DatePicker extends React.Component<Props> {
         >
           <FontAwesome name="calendar"/>
         </span>
-      </span>
+      </div>
     )
   }
 
-  render() {
-    const {
+  dateTimeProps = () => {
+    let {
```

Comment:
Our ESlint isn't properly configured yet, but we'd prefer `const` over `let` for constant values.